internal/messagequeue/messagequeue.go

```diff
@@ -34,6 +32,11 @@ const (
 	maxPriority = math.MaxInt32
 	// sendMessageDebounce is the debounce duration when calling sendMessage()
 	sendMessageDebounce = time.Millisecond
+	// when we reach sendMessaageCuttoff wants/cancels, we'll send the message immediately.
+	sendMessageCuttoff = 100
+	// when we debounce for more than sendMessageMaxDelay, we'll send the
+	// message immediately.
+	sendMessageMaxDelay = 100 * time.Millisecond
```

Comment:
This is a bit long.

Reply:
I think we can make `sendMessageMaxDelay` shorter, maybe 20ms?